website/app/domain/contracts/mailer.ts

```diff
@@ -0,0 +1,16 @@
+type EmailData = {
```

Comment:
I guess that would work too. I just like to have all the contracts of the domain in a `contracts` folder, but it's a detail/habit.

Also most of the time `contracts` is a lib by itself, because some contracts are shareable between server and client.